src/modules/cpu/kernel/crop_mirror_normalize.hpp

```diff
@@ -131,9 +132,9 @@ RppStatus crop_mirror_normalize_u8_u8_host_tensor(Rpp8u *srcPtr,
                     for (; vectorLoopCount < bufferLength; vectorLoopCount += 3)
```

Comment:
how is the AVX code and C code has the same vector loop increment

Reply:
The avx code has an increment by value (vectorIncrement = 48) and c code has an increment by 3
https://github.com/GPUOpen-ProfessionalCompute-Libraries/rpp/pull/126/files#diff-c293c511d43aee8d44e0e51e1aabf688ee9ec991096f27cd19f4dc4f99c8cb25R119

We are processing 16 RGB pixels (48 pixel values in total) per iteration in the AVX loop along width and remaining pixels along width are processed in c loop